README.md

```diff
@@ -31,12 +31,32 @@ This project aims at bringing the power of QML to the web browser.
 
 ## How to use
 #### Add the library to your web page
+Using one of the methods below, install the qmlweb JavaScript library:
 
-Install with [npm](https://www.npmjs.com/package/qmlweb),
-[Bower](http://bower.io/search/?q=qmlweb), or download directly from the
-GitHub [releases](https://github.com/qmlweb/qmlweb/releases) page.
+- [npm](https://www.npmjs.com/package/qmlweb):
 
-From that package, preload the file `lib/qt.js` in an HTML page.
+  ```
+  npm install qmlweb
+  ```
+
+- [Bower](http://bower.io/search/?q=qmlweb):
+
+  ```
+  bower install qmlweb
+  ```
+
+- GitHub [releases](https://github.com/qmlweb/qmlweb/releases):
+  ```
+  tar -xzvf v0.0.4.tar.gz
+  ```
+
+- Manually using gulp (recommended if you cloned from git):
+  ```
+  npm install
+  gulp build
```

Comment:
I originally opened this PR because it wasn't obvious where lib/qt.js came
from. It was not to bikeshed for a week about where a script tag should go.
I'd just as well reverse my addition of HTML considering how surprisingly
controversial that has been.
On Feb 25, 2016 08:08, "Сковорода Никита Андреевич" <
notifications@github.com> wrote:

> In README.md
> <https://github.com/qmlweb/qmlweb/pull/44#discussion_r54091495>:
>
> > +
> > +- [Bower](http://bower.io/search/?q=qmlweb):
> > +
> > +  ```
> > +  bower install qmlweb
> > +  ```
> > +
> > +- GitHub [releases](https://github.com/qmlweb/qmlweb/releases):
> > +  ```
> > +  tar -xzvf v0.0.4.tar.gz
> > +  ```
> > +
> > +- Manually using gulp (recommended if you cloned from git):
> > +  ```
> > +  npm install
> > +  gulp build
>
> Use npm run build instead of gulp build.
>
> It launches gulp, but does not require gulp to be installed globally for
> building qmlweb.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qmlweb/qmlweb/pull/44/files#r54091495>.
>